project-words.txt

```diff
@@ -0,0 +1,471 @@
+abernathyca
+Adriaan
+AEQUTQ
+Agan
+Alexey
+algoliasearch
+anims
+Annot
+Annots
+anonymized
+Anshul
+APFS
+appinstalled
+Applanga
+architecting
+atrule
+autoformat
+autogen
+Autogen
+backport
+backticks
+Bartosz
+baseurlissuebanner
+beforeinstallprompt
+bghg
+Bhatt
+blockquotes
+blogsidebarcount
+browserlist
+browseronly
+browserslist
+browserstack
+buble
+Buble
+Buble's
+caabernathy
+cacheable
+callouts
+Callstack
+camelcase
+Candillon
+cdabcdab
+cdpath
+Cena
+changefreq
+Changefreq
+chedeau
+Chedeau
+Clément
+clientmodules
+clsx
+codesandbox
+Configurer
+contravariance
+corejs
+creativecommons
+Csvg
+Ctqc
+customdocspath
+customfields
+customizability
+Customizability
+Daishi
+datagit
+Datagit
+Datas
+Declarators
+dedup
+deduplicated
+deploymentbranch
+DETS
+devserver
+devto
+dismissable
+Dmitry
+docgen
+docsearch
+docsify
+Docsify
+docssidenavcollapsible
+docsurl
+docu
+docusaurusconfigjs
+docusaurusplugin
+docusauruspreset
+docusuarus
+Docz
+doesn
+dogfood
+Dogfood
+dogfooding
+DXމE
+Dyzx
+easyops
+editurl
+enableupdateby
+enableupdatetime
+endi
+Endi
+Endi's
+endilie
+Endilie
+endiliey
+endobj
+endstream
+entrypoints
+ERRNAMETOOLONG
+errorboundary
+esbuild
+eslintcache
+evaluable
+exampledoc
+executionenvironment
+externalwaiting
+failfast
+FBID
+Fienny
+flac
+Flate
+Fngxꉫ
+footericon
+Formik
+FOUC
+frkv
+froms
+frontmatter
+Frontmatter
+funboxteam
+Fגwl
+gagtag
+gatrackingid
+getopts
+gitbook
+githubhost
+Gitpod
+globbing
+globby
+Globby
+GMHM
+Goss
+Goyal
+graymatter
+gtag
+Gtag
+gޝWX
+headericon
+headerlinks
+Héctor
+hideable
+Hideable
+hola
+horiz
+hostman
+Hostman
+hoverable
+HYPVT
+Iafk
+IANAD
+idempotency
+Immer
+infima
+Infima
+Infima's
+injectmanifestconfig
+inlines
+Intelli
+Interpolatable
+Jamstack
+javadoc
+JFIF
+jjka
+jkca
+jmarcey
+jscodeshift
+jssdk
+Kaszubowski
+katex
+Kato
+Kemb
+keyscan
+keytar
+Keytar
+keyvalue
+Knapen
+Kubernetes
+kukujի
+Lamana
+Lbbb
+lifecycles
+Lifecycles
+linkify
+Linkify
+lkak
+LLLI
+loadingspin
+Localizable
+localstorage
+Lorber
+Lorber's
+lowercased
+lqip
+LQIP
+lunrjs
+mapbox
+Marcey
+Marcey's
+Massoud
+mathjax
+mdast
+MDAST
+MDXAST
+MDXHAST
+metadatas
+metadatum
+metas
+Metas
+metastring
+MHTQ
+middlewares
+Middlewares
+mkcert
+mkdir
+mkdirs
+mkdocs
+MMMI
+moesif
+Moesif
+Mqypdx
+msapplication
+mycompany
+mքxz
+mڣĨY
+Nabors
+Nakagawa
+nand
+navigations
+Navigations
+navlink
+netrc
+nextjs
+Nextra
+ngryman
+Nisarag
+noflash
+noicon
+noindex
+nojekyll
+noninteractive
+noreply
+npmjs
+nprogress
+NTFS
+Nuxt
+OAGI
```

Comment:
There are a lot of these random 4-letter words. Maybe we need to ignore some other folder? Where are they coming from?

Reply:
Found some in a `.map` file and excluded those, along with removing sum upper/lowercase dups